### ISSUE TITLE
Is there any function to create channel and send message only on that channel?

### ISSUE THREAD

**Issue by ravhirizaldi** [romainrg/ratchet_client#11]
I am able to send message to all the connected clients but the problem is I don't want to send message to all the clients I just want to send the message to the clients who have subscribed the the particular channel in which the messages was sent.

can this library just that? or i must extend the function?

**Comment by jakaprata** [comment]
Yes, you can do that, just custom the code in onMessage, if you connected  to database, ajax can help you.
